src/main/reactjs/public/i18n/fi-FI.json

```diff
@@ -125,6 +126,55 @@
           "empty": "Tyhjennä valinnat"
         }
       },
+      "clerkTranslatorDetails": {
+        "buttons": {
+          "save": "Tallenna",
+          "back": "Takaisin",
+          "next": "Seuraava",
+          "cancel": "Peruuta"
+        },
+        "contactDetails": {
+          "header": {
+            "personalInformation": "Henkilötiedot",
+            "address": "Osoite",
+            "contactInformation": "Yhteystiedot"
+          },
+          "buttons": {
+            "edit": "Muokkaa"
+          },
+          "fields": {
+            "lastName": "Sukunimi",
+            "firstName": "Etunimet",
+            "identityNumber": "Henkilötunnus",
+            "street": "Katuosoite",
+            "postalCode": "Postinumero",
+            "town": "Postitoimipaikka",
+            "country": "Maa",
+            "email": "Sähköpostiosoite",
+            "phoneNumber": "Puhelinnumero"
+          }
+        },
+        "authorisations": {
+          "header": "Auktorisoinnit",
+          "buttons": {
+            "add": "Lisää auktorisointi"
+          },
+          "fields": {
+            "languagePair": "Kielipari",
+            "startDate": "Alkupvm.",
```

Comment:
Alkamis pvm

Reply:
Näin kyllä lukee designissa, mutta toi ei vastaa suomen oikeinkirjoitusta. :)

Reply:
just :D